tensorflow/python/compiler/tensorrt/test/tf_trt_integration_test_base.py

```diff
@@ -528,6 +528,20 @@ def _GetInferGraph(self, run_params, saved_model_dir):
         line_length = 160
       converter.summary(line_length=line_length, detailed=True)
 
+    print("============================================================")
+    print("Converted layers per TRTEngineOp:")
+    converted_graphdef = graph_fn.graph.as_graph_def()
+    for node in converted_graphdef.node:
+      if node.op != "TRTEngineOp":
+        continue
+      print(f"* `{node.name}`:")
+      for func in converted_graphdef.library.function:
+        if f"{node.name}_native_segment" == func.signature.name:
+          for converted_node in func.node_def:
+            print(f"\t- `{converted_node.name}`")
+          break
+    print("============================================================\n")
```

Comment:
Is this meant to stay here or was it debug?
If it stays, shouldn't it use the `logging` module instead of `print`?

Reply:
It's meant to stay. Originally I did implemented with `logging.debug` however for some reason everything using debug API is duplicated.

```
INFO:tensorflow:Linked TensorRT version: (8, 2, 5)
I0506 15:11:49.949990 139628469704512 trt_convert.py:227] Linked TensorRT version: (8, 2, 5)
INFO:tensorflow:Loaded TensorRT version: (8, 2, 5)
I0506 15:11:49.950056 139628469704512 trt_convert.py:228] Loaded TensorRT version: (8, 2, 5)
```

which makes it very difficult to read ...

Reply:
> for some reason everything using debug which makes it very difficult to read ...

What about? (untested):
```python
message = "Converted layers per TRTEngineOp:"
converted_graphdef = graph_fn.graph.as_graph_def()
for node in converted_graphdef.node:
  if node.op != "TRTEngineOp":
    continue
  message += f"* `{node.name}`:"
  for func in converted_graphdef.library.function:
    if f"{node.name}_native_segment" == func.signature.name:
      for converted_node in func.node_def:
        message += f"\t- `{converted_node.name}`"
      break
logging.debug(message)
```

Reply:
However multi-line logging might be seen as a bad practice...

Reply:
I think we are over complexifying something that is only visible when you run tests by hand and that is designed to be easy to read. I still prefer my solution

Reply:
@DEKHTIARJonathan what do you mean by the print output is "something that is only visible when you run tests by hand"?
I believe we have this guideline to choose between "print" and "log":
   if the output is not checked by the test, we should use log.

Reply:
Fixed